### ISSUE TITLE
Fix AMP validation: boilerplate AMP CSS

### ISSUE THREAD

**Issue by ChrisBAshton** [bbc/simorgh#886]
**Describe the bug**
AMP comes with a boilerplate CSS requirement; `<style amp-boilerplate` (and a `<noscript>` version) are required to make a valid AMP page.

A fix± exists here: https://github.com/BBC-News/simorgh/pull/878/commits/15712994e01df82c230af241f41f60a1bdfeee0a

±having said that, this fix appears to introduce its own warning (better than the 3 it fixes!): `The tag 'head > style[amp-boilerplate] - old variant' appears more than once in the document`. In addition, this introduces extra CSS bloat to the page - we should investigate if it is possible to avoid this.

**To Reproduce**

1. npm run build
2. npm run start
3. in a separate terminal window, do `npx amphtml-validator http://localhost:7080/news/articles/c9rpqy7pmypo.amp || true`
4. See the following error:

```
http://localhost:7080/news/articles/c9rpqy7pmypo.amp:53:2530 The tag 'head > style[amp-boilerplate] - old variant' appears more than once in the document. (see https://github.com/ampproject/amphtml/blob/master/spec/amp-boilerplate.md)
http://localhost:7080/news/articles/c9rpqy7pmypo.amp:55:30921 The mandatory tag 'noscript enclosure for boilerplate' is missing or incorrect. (see https://github.com/ampproject/amphtml/blob/master/spec/amp-boilerplate.md)
http://localhost:7080/news/articles/c9rpqy7pmypo.amp:55:30921 The mandatory tag 'noscript > style[amp-boilerplate]' is missing or incorrect. (see https://github.com/ampproject/amphtml/blob/master/spec/amp-boilerplate.md)
```

**Expected behavior**
No errors

**Additional context**
This issue is a subset of #873 and was fixed in #878, which is currently open.

- [ ] Initially labelled with ["bug"](https://github.com/BBC-News/simorgh/labels/bug)

**Comment by bcmn** [comment]
> ±having said that, this fix appears to introduce its own warning (better than the 3 it fixes!): The tag 'head > style[amp-boilerplate] - old variant' appears more than once in the document. In addition, this introduces extra CSS bloat to the page - we should investigate if it is possible to avoid this.

Exploring this issue, [AMP 2793](https://github.com/ampproject/amphtml/issues/2793) implies it is in fact related to #883, & just a very weird behaviour of the validator. However it looks like [AMP 4421](https://github.com/ampproject/amphtml/issues/4421) & the related PR was intended to fix it. 🤔

**Comment by ChrisBAshton** [comment]
Fixed in #958.
